From bbf42fbabbcaba4d8e6df25649cbf55c5b211e08 Mon Sep 17 00:00:00 2001
From: Jacek Anaszewski <j.anaszewski@samsung.com>
Date: Fri, 29 Mar 2013 12:09:10 +0100
Subject: [PATCH 0183/1302] ARM: dts: Add ak8975 device node for PQ/M0 board

This patch adds ak8975 magnetometer node and corresponding
i2c-gpio bus node for PQ/M0 board.

Signed-off-by: Jacek Anaszewski <j.anaszewski@samsung.com>
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 arch/arm/boot/dts/exynos4412-slp_pq.dts | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/arch/arm/boot/dts/exynos4412-slp_pq.dts b/arch/arm/boot/dts/exynos4412-slp_pq.dts
index 9fb3fc4..55fee8d0 100644
--- a/arch/arm/boot/dts/exynos4412-slp_pq.dts
+++ b/arch/arm/boot/dts/exynos4412-slp_pq.dts
@@ -139,6 +139,22 @@
 		};
 	};
 
+	i2c-gpio-0 {
+		compatible = "i2c-gpio";
+		gpios = <&gpy2 4 0>, <&gpy2 5 0>;
+		i2c-gpio,delay-us = <2>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		status = "okay";
+
+		ak8975@0C {
+			compatible = "ak8975";
+			reg = <0x0C>;
+			gpios = <&gpj0 7 0>;
+			position = <2>;
+		};
+	};
+
 	i2c_0: i2c@13860000 {
 		samsung,i2c-sda-delay = <100>;
 		samsung,i2c-slave-addr = <0x10>;
-- 
1.8.3.2

